pkg/router/template/template_helper_test.go

```diff
@@ -804,12 +814,85 @@ func TestClipHAProxyTimeoutValue(t *testing.T) {
 		},
 		{
 			value:    "100d",
-			expected: haproxyMaxTimeout,
+			expected: templateutil.HaproxyMaxTimeout,
 		},
 		{
 			value:    "1000h",
-			expected: haproxyMaxTimeout,
+			expected: templateutil.HaproxyMaxTimeout,
+		},
+		{
+			value:    "+-+",
+			expected: "",
+			// Invalid input produces blank output
+		},
+		{
+			value:    "1.5.8.9",
+			expected: "",
+			// Invalid input produces blank output
+		},
+		{
+			value:    "1.5s",
+			expected: "",
+			// Invalid input produces blank output
+		},
+		{
+			value:    "9223372036855ms",
+			expected: templateutil.HaproxyMaxTimeout,
+			// Exceeds the time.ParseDuration maximum
+		},
+		{
+			value:    "2147483647ms",
+			expected: "2147483647ms",
+		},
```

Comment:
```suggestion
		{
			value:    "2147483647ms",
			expected: "2147483647ms",
		},
		{
			value:    "2147483648ms",
			expected: templateutil.HaproxyMaxTimeout,
		},
```

Specifically test for the HAProxy maximum.

Reply:
ok